src/connection.rs

```diff
@@ -63,7 +63,7 @@ impl COMLibrary {
     /// `CoInitialize`s the COM library for use by the calling thread, but without setting the security context.
     ///
     pub fn without_security() -> WMIResult<Self> {
-        unsafe { CoInitializeEx(None, COINIT_MULTITHREADED)? }
+        unsafe { CoInitializeEx(None, COINIT_MULTITHREADED).ok()? }
```

Comment:
This will lose the original error code - would you mind adding the needed variant to the WMIError?

Reply:
It's slightly misleading but this calls https://docs.rs/windows-core/latest/windows_core/struct.HRESULT.html#method.ok which does retain the error code.

Reply:
🤯